src/test/scala/org/broadinstitute/dsde/rawls/jobexec/SubmissionSpec.scala

```diff
@@ -499,22 +499,6 @@ class SubmissionSpec(_system: ActorSystem) extends TestKit(_system) with FlatSpe
     }
   }
 
-  it should "502 if Cromwell can't find the workflow" in withSubmissionTestWorkspaceService { workspaceService =>
```

Comment:
These tests were removed because abort doesn't work like this anymore. Does it make sense to recreate these tests in the submission actor spec?